algorithms/ar-expsq/java/Exponentiation_by_squaring.java

```diff
@@ -0,0 +1,31 @@
+public class Exponentiation_by_squaring {
```

Comment:
Codacy found an issue: [The class name 'Exponentiation_by_squaring' doesn't match '[A-Z][a-zA-Z0-9]*'](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6356801)